src/main/java/edu/cornell/kfs/fp/document/web/struts/RecurringDisbursementVoucherAction.java

```diff
@@ -61,6 +68,44 @@ public ActionForward copy(ActionMapping mapping, ActionForm form, HttpServletReq
     	recurringForm.getPdpStatuses().clear();
     	ActionForward forward = super.copy(mapping, form, request, response);
         return forward;
+        
+    }
+    
+    public ActionForward confirmAndCancel(ActionMapping mapping, ActionForm form, HttpServletRequest request, HttpServletResponse response) throws Exception {
+        RecurringDisbursementVoucherForm recurringForm = (RecurringDisbursementVoucherForm) form;
+        RecurringDisbursementVoucherDocument recurringDV = (RecurringDisbursementVoucherDocument) recurringForm.getDocument();
+        
+        if (StringUtils.isEmpty(recurringDV.getPaymentCancelReason())) {
```

Comment:
You may want to use StringUtils.isBlank() instead of StringUtils.isEmpty() here, unless you have a need for allowing blank input.